storage/aws/aws.go

```diff
@@ -447,36 +472,43 @@ func newSpannerSequencer(ctx context.Context, spannerDB string, maxOutstanding u
 //   - IntCoord
 //     This table coordinates integration of the batches of entries stored in
 //     Seq into the committed tree state.
-//
-// The database and schema should be created externally, e.g. by terraform.
-func (s *spannerSequencer) initDB(ctx context.Context) error {
-
-	/* Schema for reference:
-	CREATE TABLE SeqCoord (
-	 id INT64 NOT NULL,
-	 next INT64 NOT NULL,
-	) PRIMARY KEY (id);
-
-	CREATE TABLE Seq (
-		id INT64 NOT NULL,
-		seq INT64 NOT NULL,
-		v BYTES(MAX),
-	) PRIMARY KEY (id, seq);
-
-	CREATE TABLE IntCoord (
-		id INT64 NOT NULL,
-		seq INT64 NOT NULL,
-	) PRIMARY KEY (id);
-	*/
+func (s *mysqlSequencer) initDB(ctx context.Context) error {
+	if _, err := s.dbPool.ExecContext(ctx,
+		`CREATE TABLE IF NOT EXISTS SeqCoord(
+			id INT UNSIGNED NOT NULL,
+			next BIGINT UNSIGNED NOT NULL,
+			PRIMARY KEY (id)
+		)`); err != nil {
+		return err
+	}
+	if _, err := s.dbPool.ExecContext(ctx,
+		`CREATE TABLE IF NOT EXISTS Seq(
+			id INT UNSIGNED NOT NULL,
+			seq BIGINT UNSIGNED NOT NULL,
+			v LONGBLOB,
```

Comment:
Worth noting that `LONGBLOB` allows values up to 4GB here which may be overkill, `MEDIUMBLOB` would allow 2^24, and `BLOB` 2^16 bytes.

Reply:
4GB is big. I actually wonder what would happen with such a large bunch of entries and we should probably check.

2^23-1 is 16 MB which is also a bit big...

Having said that: 
The default DefaultBatchMaxSize is 256, so that's 16384KB/256=64KB.
The vast majority of CT entries are smaller than this, *but*, we know of large certificates, with a lot of SAN like https://crt.sh/?id=10751627 which is 371KB. So 65KB for BLOB won't work. I've left a TODO for us to run tests and check what works well.